Packages/com.unity.inputsystem/InputSystem/InputManager.cs

```diff
@@ -1965,6 +1966,61 @@ internal void InitializeData()
             composites.AddTypeRegistration("ButtonWithTwoModifiers", typeof(ButtonWithTwoModifiers));
             composites.AddTypeRegistration("OneModifier", typeof(OneModifierComposite));
             composites.AddTypeRegistration("TwoModifiers", typeof(TwoModifiersComposite));
+
+            // Register custom types by reflection
+            RegisterCustomTypes();
+        }
+
+        void RegisterCustomTypes(Type[] types)
```

Comment:
Can we make sure to add a `ProfilerMarker` to track how long this takes?

Reply:
good idea I will add that

Reply:
@chris-massie are you concerned with startup times with this change?